jellyfin_kodi/database/__init__.py

```diff
@@ -392,6 +392,24 @@ def get_credentials():
 
     credentials['Servers'] = credentials.get('Servers', [])
 
+    # Migration for #145
+    # TODO: CLEANUP for 1.0.0 release
+    for server in credentials['Servers']:
+        if 'ManualAddress' in server:
+            server['address'] = server['ManualAddress']
+            del server['ManualAddress']
+            # If manual is present, local should always be here, but better to be safe
+            if 'LocalAddress' in server:
+                del server['LocalAddress']
+            # Functionality removed in #60
+            if 'RemoteAddress' in server:
+                del server['RemoteAddress']
```

Comment:
Not a big deal, but `RemoteAddress` does not get cleaned up if `ManualAddress` does not exist.

Ideally, this block would be at the same level as `LastConnectedMode`.